### PR TITLE
[plugin.video.tweakers] 1.1.2-1

### PR DESCRIPTION
**Pull request by skipmodea1**
Please check if this a correct pull request. I have been trouble making a correct pull request...

### CODE REVIEW

**Comment by phil65** [comment]
Please use a simple x.y.z version scheme.

File: plugin.video.tweakers/addon.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#
+# Imports
+#
+import os
+import sys
+import urlparse
+import xbmc
+import xbmcaddon
+
+reload(sys)
```
Comment (phil65) [inline plugin.video.tweakers/addon.py]: why reload(sys) ?

File: plugin.video.tweakers/addon.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon 
+	id="plugin.video.tweakers" 
+	name="Tweakers" 
+	version="1.1.2"
+	provider-name="Skipmode A1,Amelandbor,ninovanhooff">
+  <requires>
+    <import addon="xbmc.python"                 version="2.14.0"/>
```
Comment (phil65) [inline plugin.video.tweakers/addon.xml]: since you add to jarvis, this should be 2.24.0

File: plugin.video.tweakers/resources/settings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+	<!--the number after default indicates the index, index 0 is the first lvalue, index 1 is the second lvalue and so on-->
+	<setting id="preferred-video-quality" label="30300" type="enum"   lvalues="30304|30303|30302|30301"       default="3"/>
+	<setting id="debug" 		          label="30400" type="bool"                                           default="false"/>
```
Comment (phil65) [inline plugin.video.tweakers/resources/settings.xml]: please dont try to vertical-align XML attributes. I´m not sure if this is really valid XML.

**Comment by phil65** [comment]
Please convert the strings.xml files to strings.po files.
https://github.com/alanwww1/xbmc-xml2po
https://github.com/alanwww1/xbmc-xml2po/downloads

File: plugin.video.tweakers/addon.py (hunk 1 above)
Comment (skipmodea1) [inline plugin.video.tweakers/addon.py]: It seemed to fix some encoding issues for me in the past.

File: plugin.video.tweakers/addon.xml (hunk 2 above)
Comment (skipmodea1) [inline plugin.video.tweakers/addon.xml]: Ack, it should have been gotham, my bad. I'm going to close this PR now and do a gotham PR in the future.

**Comment by phil65** [comment]
just fyi, there is no need to set up another PR, just update this PR by force-pushing your updated branch.

**Comment by skipmodea1** [comment]
I'm not sure how to do that exactly, that Git-stuff is very confusing to me. Maybe something like this?:

1. <commit my changes in my github plugin.video.tweakers master>
2. git checkout jarvis
3. git pull --strategy subtree --squash plugin.video.tweakers master
4. git commit -m "[plugin.video.tweakers] 1.1.2"
5. git push origin jarvis

**Pull request closed without merging**